src/main/java/org/opentripplanner/index/model/PatternShort.java

```diff
@@ -1,26 +1,51 @@
 package org.opentripplanner.index.model;
 
-import java.util.Collection;
-import java.util.List;
-
+import com.beust.jcommander.internal.Lists;
 import org.opentripplanner.routing.edgetype.TripPattern;
+import org.opentripplanner.util.model.EncodedPolylineBean;
 
-import com.beust.jcommander.internal.Lists;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
 
 public class PatternShort {
 
     public String id;
     public String desc;
-    
+    public EncodedPolylineBean geometry;
+
+    /**
+     * Constructor for maintaining backwards compatibility
```

Comment:
This comment only really makes sense in the context of this PR. I'm not really sure it's needed. Instead a comment for the class itself might be useful.

Reply:
Refactored in 94fe2b070ea0ab38e10e88f37b271586688d48bd